pkg/cluster/task/scale_config.go

```diff
@@ -23,10 +23,11 @@ import (
 
 // ScaleConfig is used to copy all configurations to the target directory of path
 type ScaleConfig struct {
+	specManager    *spec.SpecManager
```

Comment:
I don't think the `SpecManager` should belong to the cluster spec package. It's better to move it into an independent package.

Reply:
will do this in later PR